{"name": "Ошибка 400 Bad Request при попытке сделать пустой заказ неавторизованным пользователем", "status": "passed", "description": "При попытке сделать пустой заказ неавторизованным пользователем возвращается ошибка 400 Bad Request", "steps": [{"name": "Создание заказа", "status": "passed", "parameters": [{"name": "headers", "value": "None"}, {"name": "ingredients", "value": "None"}], "start": 1737282048981, "stop": 1737282049041}], "start": 1737282048981, "stop": 1737282049042, "uuid": "2402be21-8a8f-4d4f-a101-d6607a064fe4", "historyId": "66e61c0a68623a6b501db8258dfdb75e", "testCaseId": "66e61c0a68623a6b501db8258dfdb75e", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_without_burger_not_auth_user_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "01ms32"}, {"name": "thread", "value": "7388-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}